images/win/scripts/Installers/Install-SeleniumWebDrivers.ps1

```diff
@@ -44,16 +45,39 @@ Write-Host "Chrome driver install....";
 Expand-Archive -Path "$ChromeDriverPath\chromedriver_win32.zip" -DestinationPath $ChromeDriverPath -Force;
 Remove-Item -Path "$ChromeDriverPath\chromedriver_win32.zip" -Force;
 
-Write-Host "Setting the environment variables";
+# Install Microsoft Edge Web Driver
```

Comment:
I would suggest to move this logic to Install-Edge.
Ideally, we should do this for all browsers. IMHO, it is easier to maintain

Reply:
Unfortunately we can not do that because `Install-Edge` performs before the needed folder (`C:\SeleniumWebDrivers`) is created in start of `Install-SeleniumWebDrivers`. I'd prefer to leave this workflow as is and not broke consistency